src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,42 @@
+package calculator
+
+class Calculator {
+    private fun add(a: Int, b: Int): Int {
```

Comment:
코틀린 코딩 컨벤션을 참조하셔서 메서드의 선언 순서를 변경해보시면 좋을것 같습니다.

https://kotlinlang.org/docs/coding-conventions.html#class-layout

Reply:
링크주신 문서를 읽어봤는데, 여전히 메서드 선언순서에 대해 참고할 만한 부분을 못찾았습니다.
순서에 관한 지침은 아래 내용인데
1. Property declarations and initializer blocks
2. Secondary constructors
3. Method declarations
4. Companion object

1,2,4 번 항목은 없으니 무시하고,
메서드들은 알파벳이나 가시성 순서로 정렬하지 말고 관련있는것들을 모으라고 되어있네요
제가 놓친부분이 있을까요

Reply:
>Instead, put related stuff together, so that someone reading the class from top to bottom can follow the logic of what's happening

관련있는것들은 같이 모아두되 위에서 아래로 순차적으로 메서드를 읽을 수 있도록 배치하시면 좋을것 같아 말씀드렸습니다.

외부에 노출하는 public 메서드를 가장 최상단에 선언하고,   
public 메서드 내부에서 사용되는 private 메서드를 그 하위에 선언하여   
위에서 아래로 읽었을 때 코드를 파악할 수 있도록 하는 것을 말씀드리고 싶었습니다.